File: api/Models/Subtask.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace api.Models;

public partial class Subtask
{
    [Column("id")]
    public long Id { get; set; }

    [Column("title")]
    public string Title { get; set; } = null!;

    [Column("status")]
    public string Status { get; set; } = "Not Started";

    [Column("CREATEd_at")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at")]
    public DateTime? UpdatedAt { get; set; }

    [Column("task_id")]
    public long TaskId { get; set; }

    [Column("deleted_at")]
    public DateTime? DeletedAt { get; set; }

    [Column("IsDeleted")]
    public bool IsDeleted { get; set; } = false;

    public virtual Task Task { get; set; } = null!;
}

File: mobile/app/src/main/java/com/example/purelearn/domain/model/Subtask.kt
package com.example.purelearn.domain.model

data class Subtask(
    val createdAt: String,
    val deletedAt: String,
    val id: Int,
    val status: String,
    val task: String,
    val taskId: Int,
    val title: String,
    val updatedAt: String
)